src/Automation/Interfaces/IInternalScanner.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+using System;
+
+namespace Axe.Windows.Automation
+{
+    delegate ResultsT InternalScannerCallback<ResultsT>(A11yElement element, Guid elementId);
```

Comment:
Naming conventions for type parameters in C# are either a single T or T as a prefix, not a suffix. [Read more here](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/generic-type-parameters). Personally, I found the descriptive name here more confusing than I would have found just T.